skglm/tests/test_datafits.py

```diff
@@ -29,3 +29,27 @@ def test_huber_datafit():
 
     assert_allclose(ours.coef_, their.coef_, rtol=1e-3)
     assert_array_less(ours.stop_crit_, ours.tol)
+
+
+def test_log_datafit():
+    n_samples, n_features = 10, 20
+
+    w = np.ones(n_features)
+    X, y, _ = make_correlated_data(n_samples, n_features)
+    y = np.sign(y)
+    Xw = X @ w
+
+    log_datafit = Logistic()
+    grad = log_datafit.raw_grad(y, Xw)
+    hess = log_datafit.raw_hessian(y, Xw)
+
+    np.testing.assert_equal(grad.shape, (n_samples,))
+    np.testing.assert_equal(hess.shape, (n_samples,))
+
+    exp_yXw = np.exp(-y * Xw)
```

Comment:
this can be error prone: this variable name implies that it's value is `np.exp(y * Xw)`

Reply:
yes indeed!

any suggestion?

Reply:
exp_minusyXw
it's in a test so it's not critical